### ISSUE TITLE
left marracca is thought to be in the ride side screen of the game

### ISSUE THREAD

**Issue by satanshiro** [AltoRetrato/Oneiric-Quest#3]
I cannot get the left joycon to be recognized as positioned to my left the game rezognizes height of marracas correctly but for some reason the position on x axis it does not
I am using psvr 2 adapter with steam vr

**Comment by AltoRetrato** [comment]
Satanshiro-san!

If I understood correctly, only the left controller / maraca is suffering the issue. Is that correct? What is odd is that I would expect both maracas (left and right) to behave similarly...

Anyway, I thought about creating a calibration screen instead of using some of the currently internal constants, and that could also work as a debugging tool for such issues. Unfortunately, I don't have a lot of free time at the moment to release a new version, but I'll post here when I release it so you can get an alert.

**Comment by satanshiro** [comment]
I tried multiple times but unfortunately same bug. 
I can navigate very well and intuitively in the menu but once I start a song the left maraca seems to think it’s only on the right side of the screen and doesn’t matter how much I move hand to the side it will still think it’s on the right side up and down motions seem to work. If you don’t have time I’d love to try and contribute is there a specific unreal version that I need?

**Comment by AltoRetrato** [comment]
Sorry for the late reply!

Is the issue happening only with this program?

If you want to give it a try, IIRC, I used Unreal Engine 5.1 for this [release](https://github.com/AltoRetrato/Oneiric-Quest/releases).

**Comment by satanshiro** [comment]
Yes I can use steamer games and Denice very well. Again the hand tracking works well here too in terms of holding the maracas and moving hands but in the game itself it does not register well, hopefully I’ll get the time to try and get to know the code thanks and have a nice day!
